### PR TITLE
Update one-on-one.md

### PR DESCRIPTION
**Pull request by mollyblake**
removed references to orgs that VA does not have an MOU

### CODE REVIEW

File: _employment/job-seekers/one-on-one.md [hunk 1]
```diff
@@ -60,12 +60,6 @@ You might also want to work with nonprofit organizations that have been thorough
 
         <li>
           <h3>
```
Comment (bethpotts) [inline _employment/job-seekers/one-on-one.md]: @mollyblake The first, second, and fourth piano keys now go to page not found. Could you look at the tags again to see how your removal of a piano key affected these others?

**Comment by mollyblake** [comment]
Crud. I was afraid of that ... might need some help from @awong-dev or someone

**Comment by emilyville** [comment]
@bethpotts these are correct - they won't work until they are on staging because they go into the VEC repo. But, since the tests aren't failing we know they are correct!

**Comment by emilyville** [comment]
I'm going to go ahead and merge.

**Comment by bethpotts** [comment]
@emilyville Thanks for resolving and merging!

**Comment by mollyblake** [comment]
Great! Thanks @emilyville and @bethpotts

**Pull request merged**